### ISSUE TITLE
terragrunt console in interactive mode should forward tf stdout - v0.67.x

### ISSUE THREAD

**Issue by vpham-houseloan** [gruntwork-io/terragrunt#3399]
## Describe the bug

In terragrunt v.0.67.0 - 0.67.x (0.67.4 at time of writing), the console interactive mode shows the additional STDOUT output on every character typed if not forwarding TF stdout.

## Steps To Reproduce

Without setting `--terragrunt-forward-tf-stdout` or `TERRAGRUNT_FORWARD_TF_STDOUT` environment variable:
```powershell
terragrunt console
```

## Expected behavior

The console interactive mode should either always forward TF stdout or to forward TF stdout only while typing inputs. Not sure if the latter is feasible.

Always foward TF stdout:
```
C:\some_path\a_path\sample> terragrunt console
> 1+1
2
>
```

(Alternative, if possible) Forward TF stdout only while typing inputs:
```
C:\some_path\a_path\sample> terragrunt console
> 1+1
10:45:59.156 STDOUT [.] terraform: 2
>
```
## Actual behavior

```
C:\some_path\a_path\sample> terragrunt console

10:45:51.713 STDOUT [.] terraform: >

10:45:57.279 STDOUT [.] terraform: > 1

10:45:57.957 STDOUT [.] terraform: > 1+

10:45:58.749 STDOUT [.] terraform: > 1+1

> 1+1
10:45:59.156 STDOUT [.] terraform: 2
>
```

## Nice to haves

- [X] Terminal output
- [ ] Screenshots

## Versions

- Terragrunt version: 0.67.0 - 0.67.x (0.67.4 at time of writing)
- OpenTofu/Terraform version: Terraform v1.9.5
- Environment details (Ubuntu 20.04, Windows 10, etc.): Windows 10

**Comment by yhakbar** [comment]
Thanks for calling that out, @vpham-houseloan . This should be fixed in the near future.

**Comment by levkohimins** [comment]
Resolved in [v0.67.5](https://github.com/gruntwork-io/terragrunt/releases/tag/v0.67.5) release.
